agr_java_core/src/main/java/org/alliancegenome/neo4j/repository/GeneRepository.java

```diff
@@ -691,8 +667,10 @@ default FilterComparator<T, U> thenCompare(FilterComparator<T, U> other) {
 	}
 
 	class GoHighLevelTerms {
-		@JsonProperty("class_id") private String id;
-		@JsonProperty("class_label") private String label;
+		@JsonProperty("class_id")
+		private String id;
+		@JsonProperty("class_label")
+		private String label;
 		private String separator;
```

Comment:
Why are we even using this code. As the GeneRepository will be deprecated.

Reply:
It' not being used. Will remove.